keanacobarde/on-paper
# SETTING UP FIREBASE AND AUTH - OUTSIDE TESTING OF MATERIAL UI

## Description
- Modified sample.env file with contents of needed API keys.
- Tested authentication on another repo. Authentication works.
- Performed testing on MaterialUI as a potential UI library on separate repo. Testing is successful so far. Modules have yet to be installed so web server cannot be stood just yet. 'npm install' has not been run yet.
- Stood FireBase in web console as well as Realtime Database. Sample data has not been created yet.

## Related Issue
#7 - Fully completed the setting up and testing of Firebase and Authentication. Closing ticket, even though test data has yet to be created. Data will be imported at a different time.

## Motivation and Context
This change is necessary for the functionality of this application. It cannot run without persistent data storage and authentication is handled securely.

## How Can This Be Tested?
- Pull down PR test branch. The .env file should no longer be present, as it is within the .gitignore.
- Create separate instance of .env file with API keys, if provided, install modules, and stand web server. 
- Authentication and template should work.

## Screenshots (if appropriate):
N / A 

## Types of changes
